MSAL/src/native_auth/public/MSALNativeAuthChannelType.swift

```diff
@@ -24,8 +24,14 @@
 
 import Foundation
 
+/// The possible Channel Types via which an OOB was sent
+
```

Comment:
Nit: there's an extra space that in other classes doesn't exist

Reply:
Removed